src/models/Project/index.js

```diff
@@ -1,15 +1,60 @@
 // @flow
-import createProject from './createProject'
-export * from './createProject'
-
-export type Project = {
-  id: string,
-  ownerId: string,
-  name: string,
-  slug: string,
+import { Model } from 'objection'
+import knex from '@database/index'
+import User from '@models/Token'
+
+Model.knex(knex)
+
+class Project extends Model {
+  $beforeInsert() {
+    this.createdAt = new Date()
+  }
+
+  $beforeUpdate() {
+    this.updatedAt = new Date()
+  }
+
+  static get tableName(): string {
+    return 'projects'
+  }
+
+  static get idColumn(): string {
+    return 'id'
+  }
```

Comment:
Let’s use https://babeljs.io/docs/plugins/transform-class-properties/ instead for static properties?